File: Ruby/TP8/Ej2/complex_number.rb
class ComplexNumber
  def initialize(real, imag)
    @real = real
    @imaginary = imag
  end
  def real
    @real
  end
  def imaginary
    @imaginary
  end
  def +(aux)
    ComplexNumber.new(@real+aux.real,@imaginary+aux.imaginary)
  end
end

File: Extras/TP3/ComplexNumber.java
package ar.itba.edu.TP3;

public class ComplexNumber {
    double real;
    double imaginary;

    public ComplexNumber(double real, double imaginary) {
        this.real = real;
        this.imaginary = imaginary;
    }

    public double getReal() {
        return real;
    }

    public double getImaginary() {
        return imaginary;
    }

    public ComplexNumber(double real) {
        this(real, 1);
    }
}
